generator/transit_generator.cpp

```diff
@@ -44,6 +48,23 @@ using namespace std;
 
 namespace
 {
+bool LessById(Stop const & s1, Stop const & s2)
+{
+  return s1.GetId() < s2.GetId();
+}
+
+/// \returns ref to a stop at |stops| by |stopId|.
+Stop const & FindStopById(vector<Stop> const & stops, StopId stopId)
+{
+  ASSERT(is_sorted(stops.cbegin(), stops.cend(), LessById), ());
```

Comment:
```c++
is_sorted(stops.cbegin(), stops.cend(), my::LessBy(Stop::GetId());
```
?

Reply:
There're several places where I call LessById. I see two options:
- leave method LessById() as is
- remove LessById() and call my::LessBy(Stop::GetId() in 4 places. As an advantage we remove method in that case. As a disadvantage we got a small copy paste `Stop::GetId()`

What's your opinion about that?

Reply:
Up to you. Feel free to leave as is.